src/renderEmail.js

```diff
@@ -1,6 +1,12 @@
 import ReactDOMServer from 'react-dom/server'
 
-export default function renderEmail(emailComponent) {
+export default function renderEmail(emailComponent, { renderMailchimpAttr } = {}) {
   const doctype = '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">'
-  return doctype + ReactDOMServer.renderToStaticMarkup(emailComponent)
+  let markup = doctype + ReactDOMServer.renderToStaticMarkup(emailComponent)
+
+  if (renderMailchimpAttr) {
```

Comment:
I think the MailChimp data attribute transform is really useful, but it's a separate concern from HTML email generation. I want to keep the API surface area for this library as small and generic as possible. Two ideas:

* I wonder if there's a way to get React to pass these attributes through, [`injectReactEmailAttributes`](https://github.com/chromakode/react-html-email/blob/master/src/injectReactEmailAttributes.js) style?

Alternatively:

* How about you create your own "mailchimpify" module which contains this transform, which we link to in the documentation?